### ISSUE TITLE
1.8 release breaks

### ISSUE THREAD

**Issue by prolic** [Kleidukos/get-tested#58]
Error: Can't find 'action.yml', 'action.yaml' or 'Dockerfile' under '/home/runner/work/futr/futr/setup-get-tested'. Did you forget to run actions/checkout before running your local action?

**Comment by Kleidukos** [comment]
@prolic Yes, that's why it was removed from the repository and not set as the latest version. However it seems that Dependabot does not care about either things. This is being addressed by @mmhat in https://github.com/Kleidukos/get-tested/pull/56 although we are hitting a roadblock due to https://github.com/actions/runner/issues/895

**Comment by Kleidukos** [comment]
I have opened https://github.com/dependabot/dependabot-core/issues/10587
